### PR TITLE
add 'bookmark add'

### PR DESCRIPTION
**Pull request by johnbister**

### CODE REVIEW

File: example/src/components/OperationsSection/OperationsSection.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import 'carbon-motion/dist/index.css'
 
 const OperationsSection = () => {
   const [bookmarkAnimating, setBookmarkAnimating] = useState(false)
+  const [bookmarkAddAnimating, setBookmarkAddAnimating] = useState(false)
```
Comment (kristastarr) [inline example/src/components/OperationsSection/OperationsSection.js:7]: Definitely not a blocker for merging, but for code organization purposes we have been keeping each section in alphabetical order - one of us can change it next time we are working on these same files.

**Review by kristastarr — APPROVED** [review]
Looks good!! 🔖

File: example/src/components/OperationsSection/OperationsSection.js (hunk 1 above)
Comment (kristastarr) [reply example/src/components/OperationsSection/OperationsSection.js:7]: Never mind! Let's leave it as-is.  Looking at the Carbon icon guidelines, I see those are in the same order you have here.

**Review by kristastarr — APPROVED** [review]
I merged main into this branch - everything still looks good! 😄

**Comment by johnbister** [comment]
Thanks Krista!

**Comment by johnbister** [comment]
this closes #9

**Pull request merged**